gdal/doc/source/development/rfc/rfc83_use_of_project_sponsorship.rst

```diff
@@ -0,0 +1,228 @@
+from sponsorship. This document formalizes guidelines for how that budget will
+be spent by the project and supersedes :ref:`rfc-9`
+
+There are two different type of tasks that will be funded:
+
+- Day-to-day maintenance tasks. They will be filled by several co-maintainers.
+- Tasks to achieve a specific development
+
+Day-to-day maintenance tasks
+----------------------------
+
+Scope
++++++
+
+- Bug tracker maintenance:
+
+    * Making sure that the reporter provides all details needed for
+      reproducibility.
+    * Closing reports that don't respect the guidelines, or are invalid.
+    * Tagging the issue with appropriate labels and milestones.
+    * When a fix is relevant, addressing the issue with a pull request.
+    * Adding / enhancing tests in the regression test suite should be done as
+      much as practical
+    * Tasks that are expected, or are found, to take more than a given time
+      should be confirmed with the supervisor.
+
+- Continuous integration maintenance and improvements: the project has several
+  continuous integration configurations. Some of them break regularly due to
+  external causes (network resources no longer available, change in
+  dependencies, etc.). The co-maintainer will take the necessary actions to
+  keep them in working state.
+- Review of code and documentation contributions. Such contributions are made
+  currently through GitHub pull requests. The co-maintainer will review the
+  contribution and guide the proponent through the steps to make it accepted:
+  identifying issues reported by continuous integration, asking for new tests
+  to be developed whenever possible, etc. When the contributor is not in capacity
+  to polish his/her contribution, the co-maintainer may do it him/herself.
+- Ensuring that fixes that are appropriate for backporting in stable branche(s)
+  are backported.
+- Monitoring of project communication channels (mailing list, IRC, Slack
+  channels, etc.). The co-maintainer will in particular identify issues that are
+  reported   through that means and ensures that they are captured by a
+  corresponding ticket.
+  The co-maintainer may also occasionally answer questions on usage of the
+  library and tools, but in-depth support of users is not in the scope of the
+  covered tasks   (users may use service providers for that purpose).
+- Proactive tracking of new versions of libraries that GDAL depends on, and
+  address needed changes for compilation and proper runtime behavior
+- Monitor and address reports of static and dynamic analyzers. Currently:
+  OSS-Fuzz,   cppcheck, CLang Static Analyzer, Coverity Scan, gcc&clang
+  memory/undefined behavior/etc. analyzers.
+- Documentation improvements
+- Activities related to software releases: writing the release notes,
+  issuing release candidates, addressing feedback, finalizing releases
+- For experimented contributors, help on-boarding and assist less experienced
```

Comment:
The notion of "experimented contributor" may need clarifying

Reply:
> The notion of "experimented contributor" may need clarifying

Suggestions ? Defining everything in non-subjective way is going to be very hard...

Reply:
I think this should say "experienced contributor" rather that "experimented contributor", right? That should be at least a little easier to define 😄

Reply:
"Help less experienced contributors get on board and along with GDAL" ?